### DISCUSSION TITLE
Taro 3.6.20  react-redux 9.0.4  打包钉钉/支付宝小程序， --watch模式下，正常， build模式，构建正常但钉钉开发者工具Failed to compile

### DISCUSSION THREAD

**Discussion by jsPeasant** [NervJS/taro#15020] [Q & A]
Taro 3.6.20  react-redux 9.0.4  打包钉钉/支付宝小程序， --watch模式下，正常， build模式，构建正常但钉钉开发者工具Failed to compile 
异常信息如图：
![image](https://github.com/NervJS/taro/assets/19258835/b0967289-ddd3-41bb-8248-a642f996347c)

**Comment by jsPeasant** [comment]
真的没人懂这个吗？也没人遇到过这个问题吗？

**Comment by luckyadam** [comment]
最好能有个最小复现的 demo

**Reply by jsPeasant** [reply]
有提issue哦，https://github.com/NervJS/taro/issues/15052  上面附带demo仓库
